bin/node/cli/src/service.rs

```diff
@@ -100,12 +104,16 @@ macro_rules! new_full_start {
 						keystore: builder.keystore(),
 						babe_config: sc_consensus_babe::BabeLink::config(babe_link).clone(),
 						shared_epoch_changes: sc_consensus_babe::BabeLink::epoch_changes(babe_link).clone()
+					},
+					grandpa: node_rpc::GrandpaDeps {
+						shared_voter_state: Arc::clone(&shared_voter_state),
```

Comment:
```suggestion
						shared_voter_state: shared_voter_state.clone(),
```

Reply:
Interesting, hadn't spotted that it wasn't the recommended style anymore
https://github.com/rust-lang/rust/pull/63252